lib/stats.js

```diff
@@ -5,23 +5,25 @@ var os = require('os')
 
 var platformToMethod = {
   darwin: 'ps',
-  sunos: 'ps',
-  freebsd: 'ps',
-  netbsd: 'ps',
+  sunos: 'procfile',
+  freebsd: 'procfile',
+  netbsd: 'procfile',
   win: 'wmic',
-  linux: 'ps',
-  aix: 'ps'
+  linux: 'procfile',
+  aix: 'procfile'
 }
 
 var useProcfile = fs.existsSync('/etc/alpine-release')
```

Comment:
```js
if (fs.existsSync('/etc/alpine-release')) {
  platform = 'alpine'
}
```
And then at the bottom you can just:
`require('./' + platformToMethod[platform])`